ktxspec.adoc

```diff
@@ -1014,6 +1014,45 @@ Still, `KHR_DF_PRIMARIES_BT709` / `KHR_DF_PRIMARIES_SRGB` is recommended
 for standard dynamic range, standard gamut images.
 ====
 
+==== Use of Transfer Functions
+The majority of transfer functions listed in Chapter 13 _Transfer
+functions_ of <<KDF13>>, with corresponding enumerators given in
+Chapter 5 _transferFunction_, have a linear opto-optical transfer
+function (OOTF). That is the opto-electrical transfer function
+(OETF) is the inverse of the electro-optical transfer function
+(EOTF). When a KTX file has one of these transfer functions
+
+* applications should use the EOTF to decode prior to sampling and
+  filtering;
+* mipmap generation should use the path EOTF → scaling → OETF.
+
+These transfer functions have no suffix in their enumerator names.
```

Comment:
The KDFS enumerators without a disambiguating either EOTF or OETF in the name indicate that the originating specification defines only an EOTF (such as sRGB), only an OETF (such as BT.709), or there is a linear OOTF. The proposed updated enum aliases clarify this.

Reply:
> there is a linear OOTF

I think this is the only thing KTX cares about in practice.